src/game/GameClient.java

```diff
@@ -8,12 +8,15 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.Socket;
+import java.util.Random;
 
 public class GameClient extends Thread implements GameThread {
     //private final String ServerIP = "10.24.3.121";
     private final String ServerIP = "127.0.0.1";
     private final int PORT = 1337;
     private final String myName = "HC";
+    private int botIdentifier = Math.abs(new Random().nextInt()) % 9999;
```

Comment:
Our static analysis engine detects that your pull request includes a change that a random object is created and used only once. According to [this description](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#dmi-random-object-created-and-used-only-once-dmi-random-used-only-once), it is inefficient, and if you prefer unpredictable random numbers, `java.security.SecureRandom` is strongly recommended.